docs/src/pages/getting-started/installation/dependencies.react.mdx

```diff
@@ -21,13 +21,33 @@ yarn add aws-amplify @aws-amplify/ui-react
 
 Wrap your application in `<AmplifyProvider/>` and include default styling:
 
+<Tabs>
+<TabItem title="React">
```

Comment:
Maybe "React" is not the best title for this tab item, because `Next.js` is part of React 😄  

Open for suggestions!

Reply:
![Screen Shot 2022-03-18 at 1 08 07 PM](https://user-images.githubusercontent.com/40295569/159076337-305eb778-40e3-4c43-a778-aded7180694f.png)

Chakra goes in a different way by adding a new section in the nav bar for each build tools

Reply:
I like those categories. I think it's better to be specific and say "Create React App" here, to signal that this does not work for all react build tools. In the future, we can consider making those distinctions at the nav bar level. For now:

```suggestion
<TabItem title="Create React App">
```